client/components/buttons/addButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import { TouchableOpacity, Text, StyleSheet, Animated } from "react-native";
+import { Colours, Styles } from "../../styles";
+import { ScreenProps } from "react-native-screens";
+
+type AddButtonProps = {
+  onAdd: () => void;
+};
+
+export default function AddButton({ onAdd }: AddButtonProps) {
+  const [scaleValue] = useState(new Animated.Value(0));
+  const onButtonClicked = () => {
```

Comment:
seems like you forgot to navigate here lmaoo
but the animation is soooo pretty!!!